### DISCUSSION TITLE
Plans to introduce st_force2d

### DISCUSSION THREAD

**Discussion by carlsunderman** [duckdb/duckdb-spatial#239] [General]
I know many people are asking for things, myself included but the addition of st_force2d would be nice to avoid needed to use geopandas? I've been unable to load a linestringZ into duckdb and i think the Z is the cause?

**Comment by Maxxen** [comment]
Sure, but I don't think just adding `ST_Force2D` is going to help in your case since you wont be able to even load the geometry as DuckDB `GEOMETRY` types into duckdb in the first place if the (WKB?, WKT?) data contains a z axis anyway. I think we will sooner implement support for Z and M geometries.

**Comment by Maxxen** [comment]
DuckDB has had support for Z/M and ST_Force2D for a while now.
